### ISSUE TITLE
Will it work to test customized data set using your pre-trained models?

### ISSUE THREAD

**Issue by happyCodingSusan** [mks0601/V2V-PoseNet_RELEASE#58]
Hi Gyeongsik,

Thank you for your awesome work. Especially thank your for sharing your codes!

The data set Hand2017 is collected with Intel RealSense SR300 camera as indicated by the paper "BigHand2.2M Benchmark:Hand Pose Dataset and State of the Art Analysis". 

Assume I collect a new small data set, MY_Dataset, using my own Intel RealSense SR300 camera or Intel Realsense later version SR305 camera. I am sure if I use your pre-trained model for Hand2017 data set will not work on MY_Dataset since camera calibration parameters are not exactly the same. 
Collecting a new big data set and label it is too time-consuming and it will need special device to label the ground truth of hand poses. I wonder if I can just collect a very small training data set with my camera and label it, and then fine-tune your pre-trained Hand2017 model using my small training set, do you think the fine-tuned model will work on test data set collected with my camera ?

In other words, will transfer learning work for hand pose estimation?

Huge thanks for your kind reply!

**Comment by mks0601** [comment]
Hi, I think it gonna work, and actually people use this technique. For example, they pre-train human pose estimation mode on mscoco dataset and fine-tune on posetrack dataset because joint set definition is different.

**Comment by happyCodingSusan** [comment]
Thank you very much for your quick response!!
